cmd/kubeadm/.import-restrictions

```diff
@@ -79,7 +79,6 @@
 				"k8s.io/kubernetes/pkg/util/hash",
 				"k8s.io/kubernetes/pkg/util/ipset",
 				"k8s.io/kubernetes/pkg/util/iptables",
-				"k8s.io/kubernetes/pkg/util/ipvs",
```

Comment:
i don't know if import-boss allows us to find redundant allowed imports. i think not.
but i was under the impression that by removing this one we are no longer going to need a bulk of other imports.

Reply:
Yes, I'm listing the import tree

Reply:
do you mean you verified that we can only remove this import?

Reply:
we can remove the 3 that this PR is removing

Reply:
ok, understood.
thanks